playbooks/roles/docker/tasks/uninstall.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: remove docker if running
+  raw: >-
+    docker rm -f $(docker ps -a -q --filter ancestor={{zowe_docker_image}}:{{zowe_docker_tag}})
```

Comment:
Does this work well?

Reply:
yes, tried many times locally works great, as far as we are removing same image and tag

Reply:
what happens in the case you mentioned in another block: at that level we have zowe_docker_url only not even zowe_docker_image and zowe_image_tag?

Reply:
it wont break pipeline `ignore_errors: True`
and help feature to re-run same playbook with same param locally

Reply:
we can always add additional ways to clear running container based on different conditionals and situation we encounter and enhance it

Reply:
modified as per query provided @jackjia-ibm